codesign/test/cli_test.dart

```diff
@@ -0,0 +1,7 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:test/test.dart';
+
+void main() {}
```

Comment:
Please add tests.

This is a prod tool, and it should have a robust test suite to prevent breakages.

Reply:
sure, I will add tests

Reply:
tests are now complete!